public/index.css

```diff
@@ -1,12 +1,11 @@
+@import url(~@stackoverflow/stacks/dist/css/stacks.css);
+
+/* purgecss start ignore */
 .browser-image {
     background-position: center;
     background-size: contain;
 }
 
-.browser-image.internet-explorer {
```

Comment:
Just at first glance, removing these concern me. They aren't used when parsing our default browser string, but could be used when parsing others. In addition, if we remove these from our css, we'd want to also remove these from our dependencies.

Reply:
Do we care all that much? I see this as less a general utility where users input their browserlist string, and more a fixed visual representation of Stacks' browserlist. My logic is that Stacks will never support Internet Explorer again.

If that's an unfair descoping of what this app does, I hear that. We could restore some user-input and these browser images down near the bottom of the page perhaps 🤔